mcstatus/tests/test_address.py

```diff
@@ -0,0 +1,216 @@
+import ipaddress
+from pathlib import Path
+from typing import Awaitable, Callable, TypeVar
+from unittest.mock import MagicMock, Mock, patch
+
+import dns.resolver
+import pytest
+from dns.rdatatype import RdataType
+
+from mcstatus.address import Address, async_minecraft_srv_address_lookup, minecraft_srv_address_lookup
+
+T = TypeVar("T")
+
+
+def const_coro(value: T) -> Callable[..., Awaitable[T]]:
+    """This is a helper function, which returns an async func returning value.
+
+    This is needed because in python 3.7, Mock.return_value didn't properly cover
```

Comment:
Oh yeah, I remember this pain. I used to install an async test helper to have AsyncMock.
:+1: for avoiding a dependency with a single function.